examples/tutorials/hotp/hotp_milestone_one/main.c

```diff
@@ -55,7 +55,7 @@ static void program_new_secret(hotp_key_t* hotp_key) {
     libtocksync_console_read((uint8_t*) &c, 1, &number_read);
 
     // break on enter
-    if (c == '\n') {
```

Comment:
I suspect/fear this fixes mac but breaks windows. I'm assuming the issue is that [legacy mac tools send just `\r`](https://blog.codinghorror.com/the-great-newline-schism/)? Now, though, with a DOS endpoint that sends `\r\n` this will break twice when it should only break once [previously the `\r` was just silently consumed IIUC].

Reply:
This doesn't have to do with my tockloader not sending `\n` (I can run another console recv app and verify I'm getting both `\r` and `\n`).

The issue is that they are both sent, and sent at the same time, and the loop to the next console recv is too slow to catch the `\n` (since it only reads one byte).

Reply:
I don't quite understand why being slow should matter, wouldn't the next character just be buffered and read the next time through the loop?

That said, this is a bugfix in an open tutorial, so I'm inclined to merge this for now and revisit the timing question another day.